update readme (#3)
audacity/audacity-manual: README.md

## Flow of Content

* We write the Audacity Manual at https://alphamanual.audacityteam.org

> * We get it and mangle it a little using our script, https://github.com/audacity/audacity/blob/master/scripts/mw2html_audacity/mw2html.py

Reviewer: [Nitpick/Proofreading/Question] I’m having trouble understanding what “mangle” means @Rajeta-Sarkar. Does it mean fix?
```suggestion
* We get it and mangle it a little using our script, https://github.com/audacity/audacity/blob/master/scripts/mw2html_audacity/mw2html.py
```

* That content is then pushed as a commit to the GitHub repo.
* A script on our server detects the change at GitHub and pulls the changes to serve them.

## Wiki Tricks

There is a lot of content in our manual which would be prohibitively time consuming to produce without automated tools to help us. This is particularly so of the many image maps in the manual.

> * Most of our tricks are documented at https://wiki.audacityteam.org/wiki/Magicke_Spells

Reviewer: [Nitpick/proofreading] see [line 16](https://github.com/audacity/audacity-manual/pull/3/files#r1206229006) for context. 
```suggestion
* Most of our tricks are documented at https://wiki.audacityteam.org/wiki/Magicke_Spells
```

> * The heavy lifting is done by our 'What Is That' page https://wiki.audacityteam.org/wiki/Using_WIT

Reviewer: [Nitpick/Proofreading] I think hyperlinking the page’s title would make it quicker for users to access the content. Consider doing that for line 15.
```suggestion
* The heavy lifting is done by our [What Is That page](https://wiki.audacityteam.org/wiki/Using_WIT)
```
